ui/src/version/VersionListTable.js

```diff
@@ -192,7 +192,7 @@ const VersionListTable = ({
                           <EuiLink
                             onClick={() =>
                               navigate(
-                                `/merlin/projects/${activeModel.project_id}/models/${activeModel.id}/versions/${version.id}/endpoints/${versionEndpoint.id}`
+                                `/merlin/projects/${activeModel.project_id}/models/${activeModel.id}/versions/${version.id}/endpoints/${versionEndpoint.id}/details`
```

Comment:
this is to fix the config tab hiding when clicked on the deployment environment link?

Reply:
Yes, correct.